lib/prompts/checkbox.js

```diff
@@ -96,7 +96,8 @@ Prompt.prototype.render = function (error) {
     message += chalk.cyan( this.selection.join(", ") );
   } else {
     var choicesStr = renderChoices(this.opt.choices, this.pointer);
-    message += "\n" + this.paginator.paginate(choicesStr, this.pointer, this.opt.pageSize);
+    message += "\n" + this.paginator.paginate(choicesStr, this.pointer,
+     this.opt.pageSize, this.getSeparatorOffset(this.pointer));
```

Comment:
We should pass both the pointer and the offset. A single value (pointer + offset) should work just as fine in this case.